plugins/optimization-detective/class-od-url-metric.php

```diff
@@ -208,6 +210,12 @@ public static function get_json_schema(): array {
 					'required'    => true,
 					'readonly'    => true, // Omit from REST API.
 				),
+				'eTag'      => array(
+					'description' => __( 'The ETag for the URL Metric.', 'optimization-detective' ),
+					'type'        => 'string',
```

Comment:
If using hashes for ETags, then this should have a specific `pattern`, same as is used for the `slug` argument.

Reply:
Done in e6078a9.